docs/_include/viz.rst

```diff
@@ -106,44 +106,44 @@ An example using Paraview for visualization of WEC-Sim data is provided in the `
 The **RM3_MoorDyn_Viz** example uses ParaView for WEC-Sim data visualization of a WEC-Sim model coupled with [MoorDyn](http://wec-sim.github.io/WEC-Sim/advanced_features.html#moordyn) to simulate a mooring system for the [RM3](http://wec-sim.github.io/WEC-Sim/tutorials.html#two-body-point-absorber-rm3) geometry. 
 
 
-Non-Linear Hydro Visualization in ParaView
+nonlinear Hydro Visualization in ParaView
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-When using non-linear buoyancy and Froude-Krylov Wave Excitation the paraview files also contain cell data for the bodies.
```

Comment:
minor, but should nonlinear should be capitalized

Reply:
@kmruehl The others are titles or line items so I will capitalize them. I read this more like a description (no caps) as opposed to the title of an advanced feature. I will leave it with no caps unless you feel strongly otherwise.